src/hooks/useClickableProps/__stories__/useClickableProps.stories.mdx

```diff
@@ -0,0 +1,275 @@
+import { useCallback, useRef } from "react";
+import { Canvas, Meta, Story } from "@storybook/addon-docs";
+import { Link, Tip } from "../../../storybook";
+import useClickableProps from "../useClickableProps";
+import "./useClickableProps.stories.scss";
+
+<Meta title="Hooks/useClickableProps" />
+
+# useClickableProps
+
+- [Overview](#overview)
+- [Arguments](#arguments)
+- [Returns](#returns)
+- [Feedback](#feedback)
+
+## Overview
+
+Return props for adding clickable functionality to the element except for the styles and classNames.
+
+<Canvas>
+  <Story name="Overview">
+    {() => {
+      const ref = useRef(null);
+      const onClickCallback = useCallback(() => alert("click!"), []);
+      const clickableProps = useClickableProps(
+        {
+          onClick: onClickCallback,
+          id: "clickable-id",
+          ariaHidden: false,
+          ariaHasPopup: false,
+          ariaExpanded: false
+        },
+        ref
+      );
+      return (
+        <div {...clickableProps} className="monday-storybook-use-clickable-props">
+          I act like a button
+        </div>
+      );
+    }}
+  </Story>
+</Canvas>
+
+<Tip title="Alternative">
```

Comment:
mmm not sure it's the right title in terms of UX. this is my suggestion, worth to also check this with Meirav:
```suggestion
<Tip title="Check out our component solution for this use case">
```

And then the content can be something like: 
"For most simple use cases of clickable elements, you also can use our Clickable component wrapper (Higher order component). "